### ISSUE TITLE
Pick up already caught errors

### ISSUE THREAD

**Issue by zacanger** [DataDog/browser-sdk#516]
I don't see anything in the docs on this. Is there a way to pick up already-caught errors? We just switched to Datadog from New Relic, and we were using NewRelic's `noticeError` for that. Example with NR:

```javascript
try {
  someThingThatMightError()
} catch (e) {
  someLogicToShowFriendlyErrorToUser(e)
  window.newrelic.noticeError(e, someExtraFieldsForEnrichment)
}
```

**Comment by zacanger** [comment]
Ah nevermind, it seems to be working as expected, I just didn't understand the RUM Error Tracking view.

**Comment by zacanger** [comment]
I closed this thinking I'd found it, but I was wrong. I guess what I'm looking for is the equivalent of APM's `setError` or `SetException`.

**Comment by bcaudan** [comment]
Hi @zacanger!

That is indeed a valuable feature, we'll look into it.

**Comment by zacanger** [comment]
@bcaudan thank you!

**Comment by BenoitZugmeyer** [comment]
This feature has landed in `v1.25.0`: you can now use `window.DD_RUM.addError(e, someExtraFieldsForEnrichment)`. Let us know what how it works for you!

**Comment by zacanger** [comment]
Thanks so much! We're integrating it now.
